downloads.html

```diff
@@ -5,12 +5,35 @@
 	<div class="right">
     <h1>Download</h1>
 
-    <p>2.0.0 is the latest release. The current stable version is 2.0.0.</p>
+    <p>2.0.1 is the latest release. The current stable version is 2.0.1.</p>
 
     <p>
     You can verify your download by following these <a href="https://www.apache.org/info/verification.html">procedures</a> and using these <a href="https://kafka.apache.org/KEYS">KEYS</a>.
     </p>
 
+    <span id="2.0.1"></span>
+    <h3 class="download-version">2.0.1<a href="#2.0.1"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+        <ul>
+            <li>
+                Released November 12, 2018
```

Comment:
This date is wrong, right?

Reply:
I updated the release date to tomorrow (Nov 9th), assuming we will send announce mail tomorrow.